wsmaster/che-core-api-system/src/main/java/org/eclipse/che/api/system/server/CronThreadPullTermination.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.api.system.server;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+import java.util.Collections;
+import java.util.Set;
+import org.eclipse.che.commons.schedule.executor.ThreadPullLauncher;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Terminates {@link ThreadPullLauncher}.
+ *
+ * @author Anton Korneta
```

Comment:
Looks like authorship was unintentionally copied from another class

Reply:
No, it's original authorship. I separated these changes from `hot_update` PR.